include/services/NotifierClient.h

```diff
@@ -0,0 +1,30 @@
+#ifndef OSW_SERVICE_TASKNOTIFIERCLIENT_H
+#define OSW_SERVICE_TASKNOTIFIERCLIENT_H
+
+#include "./services/OswServiceTasks.h"
+#include "./services/OswServiceTaskNotifier.h"
+
+#include <string>
+
+class NotifierClient
+{
+public:
+    NotifierClient(const std::string publisher);
+
+    std::pair<std::chrono::time_point<std::chrono::system_clock, std::chrono::seconds>, Notification> createNotification(
```

Comment:
I would recommend you to use a `typedef` or to move this into a struct - it makes reading so much easier.